frontend/src/views/camp/Story.vue

```diff
@@ -3,40 +3,45 @@ Admin screen of a camp: Displays details & periods of a single camp and allows t
 -->
 
 <template>
-  <content-card>
-    <v-toolbar>
-      <v-card-title>{{ $tc('views.camp.story.title') }}</v-card-title>
+  <content-card :title="$tc('views.camp.story.title')">
+    <template v-slot:title-actions>
+      <e-switch v-model="editing" :label="$tc('global.button.editable')"
+                class="ec-story-editable"
+                @click="$event.preventDefault()" />
+    </template>
+    <v-card-actions v-if="$vuetify.breakpoint.smAndUp">
       <v-spacer />
-      <v-btn icon
-             :href="previewUrl"
-             class="mr-4"
-             target="_blank">
-        <v-icon>mdi-printer</v-icon>
+      <e-switch v-model="editing" :label="$tc('global.button.editable')"
+                class="ec-story-editable"
+                @click="$event.preventDefault()" />
+    </v-card-actions>
+    <v-expansion-panels v-model="openPeriods" multiple flat>
+      <story-period v-for="period in camp().periods().items"
+                    :key="period._meta.self"
+                    :period="period"
+                    :editing="editing" />
+    </v-expansion-panels>
+    <v-card-actions>
```

Comment:
The stoy content can potentially get very long. Do you see any way to keep the print button at the top?

Reply:
I think this is not the first thing you want to do on this page, I don't know where to put it instead.

Reply:
Maybe in a three-dot menu next to the edit switch?